test/cpp/microbenchmarks/bm_opencensus_plugin.cc

```diff
@@ -28,6 +28,7 @@
 #include "opencensus/stats/stats.h"
 #include "src/cpp/ext/filters/census/grpc_plugin.h"
 #include "src/proto/grpc/testing/echo.grpc.pb.h"
+#include "test/core/util/test_config.h"
```

Comment:
this belongs to the other PR?

Reply:
Ok, let's rebase not that https://github.com/grpc/grpc/pull/22786 has been merged.